api/custom_urls/author.py

```diff
@@ -30,4 +30,8 @@
     # GET /author/:path_prefix/images/:imageid
     url(r'^author/(?P<path_prefix>[a-zA-Z]+)/images/(?P<id>.*)', image.Images.as_view(),
         name = 'images'),
+
+    # POST /authors
+    url(r'^authors/?$', author.QueryAuthors.as_view(),
```

Comment:
forgot to change that, i started out with a post cos i thought we would there would be query params